medusa/medusacli.py

```diff
@@ -14,41 +14,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import medusa.purge_decommissioned
+import medusa.fetch_tokenmap
+import medusa.verify
+import medusa.status
+import medusa.restore_node
+import medusa.restore_cluster
+import medusa.report_latest
+import medusa.purge
+import medusa.listing
+import medusa.index
+import medusa.download
+import medusa.config
+import medusa.backup_cluster
+from medusa.backup_manager import BackupMan
+from medusa import backup_node
+from pathlib import Path
+from collections import defaultdict
+import sys
+from click_aliases import ClickAliasedGroup
+import click
+import logging.handlers
+import logging
+import datetime
```

Comment:
**Issue:** The imports need to move back to their original location. Monkey patching has to happen before the imports.

Reply:
okay will do that